server/tests/admin.test.js

```diff
@@ -0,0 +1,126 @@
+// import chai from 'chai';
```

Comment:
This whole file is commented out so am thinking it has no business being in this PR so either delete it or ignore it